packages/ui/styles/constants/_colors.scss

```diff
@@ -26,3 +26,4 @@ $colorAccessibilityOutline: #9648bd !default;
 $colorBrand: #00bac6 !default;
 $colorShadow: rgb(53 61 73 / 20%) !default;
 $colorMenuDark: #333 !default;
+$colorCustom6: #f0f6ff !default;
```

Comment:
Would be great to come-up with a more meaningful name

Reply:
As discussed [here](https://hazelcast.slack.com/archives/C01BT11F24E/p1691670433506009?thread_ts=1691668858.529289&cid=C01BT11F24E) I would redefine `$colorOverlayBackground`

Reply:
`$colorOverlayBackground` - but we use it not (only) for overlay. Let it be `$colorPrimaryBackground`

Reply:
I mean, that we have already [`$colorOverlayBackground`](https://github.com/hazelcast/hive/blob/master/packages/ui/styles/constants/_colors.scss#L24). Let's just redefine its value to `#f0f6ff` and use it as form background.

Reply:
Merged `$colorOverlayBackground` with `$colorCustom6`.